console/console.go

```diff
@@ -189,12 +189,8 @@ func getlisttxbyaddr(addr string,page int32,pageSize int32,stype int32)( interfa
 		a:=result.(*qjson.PageTxRawResult)
 		fmt.Printf("total:%v\n",a.Total)
 		for _, t := range a.Transactions {
-			b,err:=json.Marshal(t)
-			if err!=nil{
-				fmt.Println("getlisttxbyaddr err:",err.Error())
-				return nil, err
-			}
-			fmt.Printf("%s\n",string(b),)
+
```

Comment:
1.) What's the purpose of your function `getlisttxbyaddr`,  If you only want to log your behavior please using `log` instead of using `fmt.Printf` directly,  if you have another side-effect to print out STDOUT, Please refactor your function, to separate responsibility from the function.
2.) Please use CamelCase style.